test/unit/test_devices/test_eos_device.py

```diff
@@ -31,17 +31,17 @@ def test_config(self):
         result = self.device.config(command)
 
         self.assertIsNone(result)
-        self.device.native.config.assert_called_with([command])
```

Comment:
This test case should be duplicated to test passing both a single command as a string and passing a list of commands